outlines/models/transformers.py

```diff
@@ -90,6 +90,6 @@ def transformers(model_name: str, device: Optional[str] = None, **model_kwargs):
     from transformers import AutoModelForCausalLM
 
     model = AutoModelForCausalLM.from_pretrained(model_name, **model_kwargs)
-    tokenizer = TransformersTokenizer(model_name)
+    tokenizer = TransformersTokenizer(model_name, **model_kwargs)
```

Comment:
Are model kwargs and tokenizer kwargs necessarily the same?

Reply:
No they are not, from the [docs](https://huggingface.co/docs/transformers/model_doc/auto#transformers.AutoTokenizer), there are different kwargs for tokenizer. In this case the `trust_remote_code` is shared.